### DISCUSSION TITLE
Private list members

### DISCUSSION THREAD

**Discussion by x0aa7i** [tweepy/tweepy#1896] [Questions] [answered]
Does `API.get_list_members()` support private lists? (It's not mentioned in docs.)
I can only get members of public lists using the code below
Private lists return `34 - Sorry, that page does not exist.` 
All lists I tested are mine, btw.

```python
auth = tweepy.OAuth2BearerHandler(BEARER_TOKEN)
api = tweepy.API(auth)

members = api.get_list_members(list_id=list_id)
```

**Comment by x0aa7i** [comment]
Using `tweepy.OAuth1UserHandler` to authenticate as a developer account, fixed that.

**Comment by Harmon758** [comment, marked as answer]
Yes, you do need to use OAuth 1.0a User Context (with `OAuth1UserHandler`) and authenticate as a user that has access to the lists in question, as OAuth 2.0 Bearer Token is an app-only authentication method.

**Comment by Harmon758** [comment]
@idclevxx Yes, that method and endpoint work fine for me. Double check that your credentials are correct.
